doc/source/programs/vector_common_options.rst

```diff
@@ -51,6 +51,7 @@ All GDAL OGR command line programs support the following common options.
     A value of OFF will disable all debug messages.
     Another value will select only debug messages containing
     that string in the debug prefix code.
+    Multiple prefixes need to be separated by `|`.
```

Comment:
Not really. The doc of the CPLDebug() function mentions "... Otherwise only messages whose category appears
 somewhere within the CPL_DEBUG value are displayed (as determined by strstr())."  . The sentence "Another value will select only debug messages containing hat string in the debug prefix code." should rather be amended to reflect that.

Reply:
For future reference, the before-mentioned docs is available at

https://github.com/OSGeo/gdal/blob/2cb831638ea63c3cf38c148b46b4887869231f3d/port/cpl_error.cpp#L579-L602

Reply:
The above has been coped and rephrased